openmoc/materialize.py

```diff
@@ -556,7 +566,8 @@ def compute_sph_factors(mgxs_lib, max_sph_iters=30, sph_tol=1E-5,
         fsrs_to_domains[fsr] = domain.getId()
         domains_to_fsrs[domain.getId()].append(fsr)
 
-        if domain.isFissionable():
+        if (sph_domains == None and domain.isFissionable()) or \
```

Comment:
`if not sph_domains`

Reply:
done. Adapted rest of routine to use same style (not instead of ==None)